addon/components/session-overview.hbs

```diff
@@ -241,14 +241,19 @@
               {{else}}
                 <label>
                   {{#if @session.hasPostrequisite}}
-                    {{#link-to "session.index" (await @session.postrequisite.course) (await @session.postrequisite)}}
-                      {{fa-icon  icon="external-link-square-alt"}} {{t "general.duePriorTo"}}:
-                    {{/link-to}}
+                    <LinkTo @route="session.index" @models={{array (await @session.postrequisite.course) (await @session.postrequisite)}}>
```

Comment:
`@models` is super slick with the array helper. I LIKE this code.

Reply:
me too! found this here when revisiting the online docs https://guides.emberjs.com/release/routing/linking-between-routes/#toc_multiple-dynamic-segments

Reply:
I don't want to know **how** the magic works!